documentation/01_tutorial/04-3-groundwork.html.md

```diff
@@ -37,12 +37,13 @@ First up, let's create a simple menu. Eventually, we'll want a fancy `MenuState`
 
 	As before with `FlxText`, the import should be added automatically. If not, you can also add missing imports via the light bulb that should appear after saving the file when you have your cursor over `FlxButton`.	
 
-2. Now, type the following on the lines before `super.create();` within the `create()` method:
+2. Now, type the following on the lines after `super.create();` within the `create()` method:
```

Comment:
Doesn't really matter.

Reply:
I put both, after and before, is that ok?